docs/tutorial/fdcurves.rst

```diff
@@ -71,3 +71,21 @@ and distance from such an ensemble using::
 
     f = fd_ensemble.f
     d = fd_ensemble.d
+
+Baseline Correction
+-------------------
+
+.. note::
+    By default, FD curves are constructed using the force magnitude :math:`F = \sqrt{F_x^2 + F_y^2}`. However, baseline
+    correction in Bluelake is only calculated for the x-component :math:`F_x`. Therefore, FD curves with baseline
+    correction applied are constructed with only the x-component rather than the full magnitude and may not be directly
+    comparable to the corresponding uncorrected FD curve.
+
+    Additionally, baseline-correded FD curves are read directly from the source HDF5 file. Therefore, any data manipulations
+    (such as offset correction) that were previously applied to the calling instance will no longer be present.
```

Comment:
This sentence may be a bit tricky to understand for some. I'm also not a huge fan of the word "manipulation". It sounds evil :D. Maybe something along the lines of:

```Therefore, any data processing previously applied to the `Fd Curve` used to obtain the baseline corrected curve is lost.```

That said, I do like how prominent the note is though.